bin/common.sh

```diff
@@ -59,8 +59,7 @@ function check_java_version() {
   # JDK 8u151 version fixed a number of security vulnerabilities and issues to improve system stability and security.
   # https://www.oracle.com/java/technologies/javase/8u151-relnotes.html
   if [ "$JVM_VERSION" -lt 8 ] || { [ "$JVM_VERSION" -eq 8 ] && [ "${jvmver#*_}" -lt 151 ]; } ; then
```

Comment:
hi @pan3793 Thank you for your interest in the Gravitino project.

I think this condition needs split to be 2.
1. JVM_VERSION must greater than 8, otherwise exit.
2. If JVM_VERSION less than 8.151, then print warn message.

Reply:
thanks for checking, updated as suggested.